commands/logs.go

```diff
@@ -11,13 +11,16 @@ import (
 
 	"github.com/mattermost/mattermost-server/v5/mlog/human"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
 	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
 )
 
 var LogsCmd = &cobra.Command{
 	Use:   "logs",
 	Short: "Display logs in a human-readable format",
+	Long:  "Display logs in a human-readable format. The \"--format\" flag cannot be used here.",
```

Comment:
Suggesting a small change for clarity:

```suggestion
	Long:  "Display logs in a human-readable format. As the logs format depends on the server, the \"--format\" flag cannot be used with this command.",
```

Reply:
fixed